### ISSUE TITLE
qgsmapcanvas.maprenderer deprecated

### ISSUE THREAD

**Issue by fgillen01** [nilsnolde/ORStools#34]
QGIS Version 2.18.4
OSM Tools v.0.3.1
'QuickMapServices' v.0.18.7

Open up QGIS > Add Google Roads Layer from QuickMapServices > CRS = WGS / Pseudo Mercator EPSG:3857 > Open up OSM Tools plugin and add API key > Select Default settings for 'Accessibility Areas' > Select 'Map now!' and pick place on map > Nothing happens 

Below is the warning I get from the Log Messages Panel

![image](https://user-images.githubusercontent.com/13051473/30800573-57e1491a-a1d8-11e7-8c0c-bd634f166357.png)

**Comment by nilsnolde** [comment]
Hi Finbar,

thanks for being more detailed. QuickMapServices shouldn´t have anything to do with it.

Again, I tried with the same QGIS version and it works. Also if the map canvas is on epsg:3857. 

However, googling the issue, it appears that your error is correct: Geolicious/qgis2leaf/issues/#107. Which is definitely weird, it should be deprecated on my version as well.

Anyways, I´ll correct it in the next version, 0.3.3. Bummer, I just pushed 0.3.2, but that´s on me, could´ve researched before..
